### PR TITLE
[WIP] Add standard about testing

### PR DESCRIPTION
**Pull request by AuroreM**

### CODE REVIEW

**Comment by bam-bot-tech** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/test-standard/project-standards/technical-agility/react-native-test.s.md">react-native-test.s.md</a>**: Does not seem to be included in the root readme
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **<a href="https://github.com/bamlab/dev-standards/blob/test-standard/project-standards/technical-agility/react-native-test.s.md">react-native-test.s.md</a>**: Does not seem to be included in the root summary
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

File: project-standards/technical-agility/react-native-test.s.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+  - 4) The presentational components are tested with a snapshot. It avoids UI regression.
+  - 5) The services are tested. It helps to decouple the code.
+
+
+## Bad Examples
+*TBD*
+
+## Good Examples
+1) A reducer and its test (~2min):
+```
```
Comment (tychota) [inline project-standards/technical-agility/react-native-test.s.md]: ` ``` javascript`

File: project-standards/technical-agility/react-native-test.s.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+      SIMSerialNumber: '89101214',
+    },
+  };
+  const nextState = reducer(initialState, action);
+  expect(nextState.SIMSerialNumber).toEqual('89101214');
+});
+```
+
+A selector and its test (~2min):
+```
```
Comment (tychota) [inline project-standards/technical-agility/react-native-test.s.md]: ` ``` javascript`

File: project-standards/technical-agility/react-native-test.s.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+      },
+    },
+  };
+  expect(SIMSerialNumberSelector(state)).toEqual('1234');
+});
+```
+
+2) A saga and the test of the order execution:
+
+```
```
Comment (tychota) [inline project-standards/technical-agility/react-native-test.s.md]: ` ``` javascript`

File: project-standards/technical-agility/react-native-test.s.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+// Saga
+export function* cancelPlan(action) {
+  const uuidPlan = yield select(nextPlanUuidSelector);
+  try {
+    yield call([tripica, 'deleteSandBoxByUuid'], uuidPlan);
```
Comment (tychota) [inline project-standards/technical-agility/react-native-test.s.md]: Pas de client ! c'est public

**Review by yleflour — APPROVED** [review]

**Comment by yleflour** [comment]
Merging as first version

**Pull request merged**